test/buy-tshirt.spec.ts

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable protractor/no-describe-selectors */
+import { browser, by } from 'protractor';
+import { config } from '../protractor/local.config';
+
+
+describe('Buy a t-shirt', () => {
+
+  beforeEach(async() => {
```

Comment:
It seems your configuration of eslint isn't working, please check it out
For example here, should be:
```suggestion
  beforeEach( async () => {
```